src/engraving/dom/excerpt.cpp

```diff
@@ -348,10 +348,7 @@ void Excerpt::createExcerpt(Excerpt* excerpt)
     MeasureBase* scoreMeasure = score->first();
 
     if ((!scoreMeasure || !scoreMeasure->isVBox()) && !masterMeasure->excludeFromOtherParts()) {
-        Score::InsertMeasureOptions options;
-        options.addToAllScores = false;
-
-        score->insertMeasure(ElementType::VBOX, scoreMeasure, options);
+        score->insertMeasure(ElementType::VBOX, scoreMeasure);
```

Comment:
I would prefer to use `insertBox` in those places where the type is explicitly a box type anyway